### DISCUSSION TITLE
Show Addon airports shows All MSFS stock airports

### DISCUSSION THREAD

**Discussion by vin747** [albar965/littlenavmap#917] [Q&A]
Hi Alex,

I've had this problem since last night after a scenery db load process.. It keeps showing all stock airports as addons.. I was on 2.7.9, went back to 2.6.19 (stable), still no luck..  upgraded to 2.7.11 and still same issue.. can you pls advise? I'm on SU9, not Su10 beta. 

Your manual says 2.6.18 has addressed this.. but looks like problem is still there..

Thanks for your amazing gift to the community btw.. have been a happy LNM user for many years now..

**Comment by vin747** [comment]
After every version switch, I did reload scenery DB.. also, not using navigraph if that matters.. so I have that option 'do not use navigraph' checked..

**Reply by albar965** [reply]
Sorry, I need more information.

Asobo did not change anything recently. The problem should not appear on either LNM version and I do not see this here neither on my flying computer nor on test systems.

What MSFS installation do you have (Steam, MS Boxed or MS download)?
Do you use a scenery management tool? Can you find a folder `.../Microsoft.FlightSimulator_8wekyb3d8bbwe/LocalCache/Packages/Official/OneStore/fs-base-genericairports` ?

You can reload the scenery library and then send me your log file (https://www.littlenavmap.org/manuals/littlenavmap/release/2.6/en/FILES.html#logs) to https://albar965.github.io/contact.html which might help me to identify the problem.

Alex

**Comment by vin747** [comment]
Have sent you an email with the log files. I'm on MSFS steam version, SU9. 
LNM version is 2.7.11.

Yes, I can find fs-base-genericairports folder in D:\MSFS\Packages\Official\Steam\fs-base-genericairports.

Also, the 'show addon' airports feature was working perfectly fine for last many months until last night when I added some sceneries thru addon linker and then reloaded scenery DB in LNM..

**Reply by albar965** [reply]
Thank you for the log files. Got them.

I checked again with 2.6.19 and 2.7.11.develop and see no issues. I have no idea what's wrong in your case. I also did not get any other reports about this. Looked through your logs and see no apparent issues.

Maybe the scenery add-on linker adds (symbolic) links for some folders thus confusing LNM.

Are all airports marked as add-on or many? 
The latter case might be caused by Orbx add-ons which often add small airport updates correcting only the elevation. You can exclude Orbx folders in LNM options from add-on detection if this is the case.

As a workaround you can try to add `fs-base`,`fs-base-genericairports` and `fs-base-nav` to the add-on exclusion list in LNM options on page `Scenery Library Database` at the lower list. Not sure if this works if these folders are links.

Alex

**Comment by vin747** [comment]
Hi,

It is showing all 40,000 airports I think.. way more than ORBX.. its
showing all my addon airports + all stock MSFS airports.. see pic below..

[image: image.png]

nobody can have this many addon airports!

Exclude path didnt solve the issue.. not sure what to do now..

I even tried deleting the .sql files in roaming folder and let it rebuild..
still no luck..

Vinod.

**Reply by albar965** [reply]
You can disable the add-on highlighting in options on page `Map Display` as another workaround. Uncheck `Highlight add-on airports` there to get rid of the yellow circles. 

Or some settings are broken in LNM. Temporarily rename the `ABarthel` here `C:\Users\YOURUSERNAME\AppData\Roaming\ABarthel` which is like a total reset.

No idea what can cause this. The only explanation is the scenery manager doing something strange to the paths.

Alex

**Comment by vin747** [comment]
The full reset didn't help either.  Only big change to the sim I did
recently was install gsx which does something to all stock airports. So not
sure if gsx did something to the paths.

Vinod.

**Reply by albar965** [reply]
Exclude the GSX folder from add-on detection. The add-on probably adds a lot of airports which modify the stock ones and these are wrongly recognized as add-ons.

Like the Orbx regions which add altitude corrections to all airports and are falsely recognized as add-on in LNM. This is more or less described here: https://www.littlenavmap.org/manuals/littlenavmap/release/2.6/en/OPTIONS.html#scenery-library-database
Alex

**Comment by vin747** [comment]
[image: image.png]

Have excluded GSX master folder and fs-base-genericairports folder, but
still same issue..

Is it possible to extract or export the addon airport list from LNM? I'm
sure it must be storing it somewhere in SQL.. should be readable in SQL
viewer. Can we manually edit that list then and then reload it back?

Vinod.

**Comment by vin747** [comment]
I think for the time being, I've found a good workaround.. I'm using the
addon linker's in-built feature of showing a map with all the active addon
airports.. I can use this to select my city pair and then go to LNM for
flight planning and other stuff..

You can close the ticket.. thank you for your patience and your awesome
flight planning tool!

Vinod.

[image: image.png]

**Reply by albar965** [reply]
> Is it possible to extract or export the addon airport list from LNM? I'm sure it must be storing it somewhere in SQL.. should be readable in SQL viewer.

You can open the LNM databases with any SQL viewer. You can also change the content but this might cause issues like SQL Exceptions. I added a bit of documentation in the `create_*.sql` scripts here: https://github.com/albar965/atools/tree/master/resources/sql/fs/db

> I think for the time being, I've found a good workaround.. I'm using the addon linker's in-built feature of showing a map with all the active addon airports.. I can use this to select my city pair and then go to LNM for flight planning and other stuff.

Good you found a solution. I suppose you use this [ add on linker](https://de.flightsim.to/file/1572/msfs-addons-linker), right? I should really run some tests with this one.

Alex
